cpp/llm_chat.cc

```diff
@@ -65,13 +65,12 @@ std::unique_ptr<Tokenizer> TokenizerFromPath(const std::string& _path) {
       std::filesystem::path merges_path = path / "merges.txt";
       std::filesystem::path vocab_path = path / "vocab.json";
       std::filesystem::path added_tokens_path = path / "added_tokens.json";
-      if (std::filesystem::exists(merges_path) && std::filesystem::exists(vocab_path)) {
+      if (std::filesystem::exists(merges_path) &&
```

Comment:
Thanks for the fixing, a comment posted here for future reference:

The behavior of which tokenizer files to use should actually be specified by `mlc_chat_cli.json`.
Ideally, if we want to use `tokenizer.json` we should specify 
```
  "tokenizer_files": [
      "tokenizer.json"
  ]
```
in `mlc_chat_cli.json`.

I'll further refactor this function to follow user-specified `tokenizer_files` in a later PR. I'm okay with the fix here and we can merge this first.